wpiutil/src/main/java/edu/wpi/first/util/RuntimeLoader.java

```diff
@@ -48,6 +49,73 @@ public static void loadLibrary(String libraryName) throws IOException {
     }
   }
 
+  /**
+   * Load a single library from a list of extracted files.
+   *
+   * @param libraryName The library name to load
+   * @param extractedFiles The extracted files to search
+   * @throws IOException If library was not found
+   */
+  public static void loadLibrary(String libraryName, Iterable<String> extractedFiles)
```

Comment:
This stays with CombinedRuntimeLoader. This would never be called from a non tool project

Reply:
The reason I moved it was it calls getLoadErrorMessage, and moving it allowed removing the CombinedRuntimeLoader version, which was also out of date.